cmd/kube-controller-manager/app/controller_manager_test.go

```diff
@@ -73,9 +75,7 @@ func TestIsControllerEnabled(t *testing.T) {
 
 	for _, tc := range tcs {
 		actual := IsControllerEnabled(tc.controllerName, sets.NewString(tc.disabledByDefaultControllers...), tc.controllers...)
-		if actual != tc.expected {
-			t.Errorf("%v: expected %v, got %v", tc.name, tc.expected, actual)
-		}
+		assert.Equal(t, actual, tc.expected, fmt.Sprintf("%v: expected %v, got %v", tc.name, tc.expected, actual))
```

Comment:
Seems wrong usage here, should change the position of expected and actual.

Reply:
o, let me check. Thanks!